vscode_projects/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
+This extension provides a quick introduction on how to use the new Python pluging interface.
```

Comment:
this string appears in the settings and should serve as longer description of the plugin